lms/static/certificates/sass/_components.scss

```diff
@@ -0,0 +1,641 @@
+}
+
+.header-app {
+
+    .logo {
+        @extend %reset-link;
+    }
+}
+
+// ------------------------------
+// #BANNER
+// ------------------------------
+.banner {
+
+    .message-copy {
+        margin-bottom: 0;
+    }
+}
+
+// message to user-centric banner
+.wrapper-banner-user {
+    box-shadow: inset 0 rem(2) rem(2) 0 palette(grayscale, black-t);
+    background: palette(grayscale-cool, x-dark);
+}
+
+.banner-user {
+
+    .message-title {
+        color: palette(grayscale, white-t);
+        font-weight: font-weight(semi-bold);
+    }
+
+    .message-copy {
+        color: palette(grayscale, white);
+    }
+
+    .message-actions .action {
+        display: inline-block;
+        vertical-align: middle;
+        margin-right: spacing-horizontal(small);
+        font-size: font-size(mid-large);
+
+        &:last-child {
+            margin-right: 0;
+        }
+    }
+}
+
+// ------------------------------
+// #INTRODUCTION
+// ------------------------------
+.introduction {
+    margin-bottom: spacing-vertical(base);
+    text-align: center;
+}
+
+// ------------------------------
+// #ACCOMPLISHMENTS
+// ------------------------------
+// base accomplishment
+.accomplishment {
+
+    .accomplishment-type-symbol {
+        @extend %img-cropped;
+    }
+}
+
+// main accomplishment
+.accomplishment-main {
+    background: palette(grayscale-cool, x-trans);
+}
+
+// brief accomplishment
+.accomplishment-brief {
+    @extend %depth-card;
+
+    .accomplishment-type-symbol, .accomplishment-details {
+        display: inline-block;
+        vertical-align: middle;
+    }
+
+    .accomplishment-type-symbol {
+        @include size(20%);
+    }
+
+    .accomplishment-details {
+        width: 70%;
+    }
+
+
+    .accomplishment-summary {
+        margin-bottom: 0;
+    }
+}
+
+// ------------------------------
+// #RENDERING
+// ------------------------------
+%rendering-accomplishment-line {
+    display: block;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+%rendering-section {
+    margin-bottom: spacing-vertical(base);
+    border-bottom: rem(2) solid palette(grayscale-cool, x-light);
+    padding-bottom: spacing-vertical(base);
+
+    &:last-child {
+        border: none;
+        padding-bottom: 0;
+    }
+}
+
+%rendering-hd-section {
+    margin-bottom: spacing-vertical(mid-small);
+    border-bottom: rem(2) solid palette(grayscale-cool, x-light);
+    padding-bottom: spacing-vertical(x-small);
+    color: $cert-base-color;
+    text-align: center;
+}
+
+.accomplishment-rendering {
+    @extend %depth-card;
+    position: relative;
+    top: -(spacing-vertical(base));
+    border-top: rem(4) solid $cert-base-color;
+
+    // type
+    .accomplishment-type {
+        text-align: center;
+        margin-bottom: spacing-vertical(base);
+    }
+
+    .accomplishment-type-label, .accomplishment-type-symbol {
+        display: block;
+    }
+
+    .accomplishment-type-label {
+        width: span(10);
```

Comment:
NOTE: new susy grid-management syntax. ``span(*)`` is a susy function that will do grid math and give us the width of columns separated from the actual laying out of the columns. Handy for when you want control of the layout, but want the same calculations to base that off of.

See http://www.sitepoint.com/creating-responsive-grid-system-susy-breakpoint/ and http://susydocs.oddbird.net/en/latest/.